src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -195,6 +204,31 @@ public class UTIL_ACCT_Naming {
         return accountNamingFormat;
     }
 
+    /*******************************************************************************************************
+    * @description This calls the UTIL_SortContacts to sort Contacts in alphabetical order.
+    * @param cons a list of Contacts that needs to be alphabetized.
+    * @return List<Contact>.
+    */
+
+    public static List<Contact> sortContacts(Contact[] cons) {
+        List<UITL_SortContacts> sortedContacts = new List<UITL_SortContacts>();
+        List<Contact> finalSortedContacts = new List<Contact>();
+
+        UITL_SortContacts sortCons;
+
+        for (Contact c : cons) {
+            sortCons = new UITL_SortContacts(c);
+            sortedContacts.add(sortCons);
```

Comment:
You could just get rid of line 217 above. This could be consolidated to `sortedContacts.add(new UTIL_SortContacts(c));`